### PR TITLE
Feature/openpose singleshot

### PR DESCRIPTION
**Pull request by LoyVanBeek**
Fixes https://github.com/tue-robotics/tue_robocup/issues/784

### CODE REVIEW

File: scripts/openpose.service [hunk 1]
```diff
@@ -17,4 +17,4 @@ ExecStart=/bin/bash -c 'source /home/amigo/.tue/setup.bash \
         _net_output_height:=368 \
         _model_folder:=/home/amigo/openpose/models/ \
         __ns:=hero/openpose \
-        image:=/hero/head_rgbd_sensor/rgb/image_raw'
+        image:=/ignore/images/topic/because/we/want/only/rosservice/requests'
```
Comment (ar13pit) [inline scripts/openpose.service]: @LoyVanBeek You know how hackers are identified based on the things they leave in their, this is how you will be caught :rofl:

**Comment by LoyVanBeek** [comment]
> You are way behind. Because the service has been renamed to hero2-openpose.service. And this is too dirty fix.

And where would I find the proper file? 
What would be a better way? I prefer to not change the image_recognition code itself.

**Comment by MatthijsBurgh** [comment]
The openpose node should be able to run both topic and service mode. Maybe it is already in there.

**Comment by LoyVanBeek** [comment]
> The openpose node should be able to run both topic and service mode. Maybe it is already in there.

It is capable of both. I just point the topic to somewhere else so only the service  is used.

**Comment by MatthijsBurgh** [comment]
> both. I just point the topic to somewhere else s

It should be a parameter, don't let it listen to a dead topic.

**Comment by LoyVanBeek** [comment]
Matthijs, I'm getting quite done with this. 
- The topic is already a parameter so we can do stuff like this.
- do not want to do a new release of image_recognition
- we have a tournament in a few weeks
- this is 100% self-explanatory in the config of a robot where I think it should be.

**Comment by ar13pit** [comment]
I have added a topic mode parameter in tue-robotics/image_recognition#93. Once that PR is merged, then this can be merged.

**Comment by ar13pit** [comment]
@MatthijsBurgh My changes in this PR have been agreed upon by both @LoyVanBeek and @reinzor. So this is GTM.

**Comment by MatthijsBurgh** [comment]
> @MatthijsBurgh My changes in this PR have been agreed upon by both @LoyVanBeek and @reinzor. So this is GTM.

If they agree, they should approve. I will do it for now.

**Pull request merged**